swagger/swagger-listing.yaml

```diff
@@ -188,9 +188,9 @@
           - "Listing"
         responses:
           200:
-            description: "An array of listing"
+            description: "An array of PreferenceBucket"
             schema:
-              $ref: "#/definitions/ApplicationResult"
+              $ref: "#/definitions/PreferenceBucket"
```

Comment:
would it make sense to return an array of `PreferenceBucket` here? It seems like that would be the most reusable way of accomplishing this.